src/app/layout/footer/footer.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../styles.colors.scss';
```

Comment:
Instead of `@import` you should use `@use`. `@import` is close to be deprecated and it loads a stylesheet file every time it's used, while `@use` directly reuses the file that is already loaded.
```
@use '../../../styles.colors.scss' as colors;

.page-footer {
  background-color: colors.$primary-color;
}
```

Reply:
Thank you. Good tip